### DISCUSSION TITLE
Student ID card has name & ID and date on oppisite sides

### DISCUSSION THREAD

**Discussion by Couleslaw** [community/community#110561] [GitHub Education] [answered]
### Hi everyone,

I am having an issue with uploading proof of my academic status. My student ID card has my name + student ID and expiration date on opposite sides. Since you can upload only 1 photo, I am not able to give sufficient proof. I can either submit a photo with my name + student ID, or a photo with the date. Is there some way of resolving this?

**Comment by samus-aran** [comment, marked as answer]
Hi @Couleslaw 👋 

In order to qualify for the Github education program, we must be able to verify that you are currently and actively enrolled in your academic program. Dated documents that will most help you to be most successful in getting approved are the following:

- Picture of your school ID with the current enrollment date
- Dated Class schedule / Transcript
- Dated Affiliation / enrollment verification later

For countries utilizing non-standard academic calendars, you may need to capture the original document beside one with a converted date. **You may include multiple documents in your image, so long as they are legible.** So you could take a photo of your student ID card along side a dated class schedule.

Hope that helps 👍

**Comment by Shrikrishna10** [comment]
Hey it doesnt take my file upload at all keeps throwing this random triangle with no text when i try to apply
![image](https://github.com/user-attachments/assets/e44ce938-762b-4264-a0fc-43212056c5a8)

**Reply by saxenaprajjwal** [reply]
Yeah, I have the same issue. I have taken these two steps (two-factor authentication and billing address) and also logged out of GitHub quite a few times, but this triangle keeps appearing after I upload my ID. 
![Screenshot 2025-07-07 194544](https://github.com/user-attachments/assets/13fffcf9-f980-4a13-879b-0fe5a2788031)
